README.md

```diff
@@ -13,7 +13,7 @@
 
 etcd is a distributed, consistent key-value store for shared configuration and service discovery, with a focus on being:
 
-* *Simple*: well-defined, user-facing API (gRPC)
+* *Usable*: well-defined, user-facing API (gRPC)
```

Comment:
I think we want to say the API is simple, not the code itself is "simple". We know that the code is not simple since it involves raft for example. By the way, etcd's sloc is around 80-90k not 220k. We probably should not count its dependencies.